### ISSUE TITLE
Missing POM file in lookup URL

### ISSUE THREAD

**Issue by ncreep** [coursier/coursier#456]
Hi,

I'm trying to pull the [scala-maxmind-iplookups](https://github.com/snowplow/scala-maxmind-iplookups) dependency. I'm using coursier `1.0.0-M15-5`, with the following `build.sbt` file:
```scala
scalaVersion := "2.11.8"

resolvers += "SnowPlow Repo" at "http://maven.snplow.com/releases/"
resolvers += "Twitter Maven Repo" at "http://maven.twttr.com/"

libraryDependencies += "com.snowplowanalytics"  %% "scala-maxmind-iplookups"  % "0.2.0"
```

When resolving the dependency I get the following:
```
Downloading https://repo1.maven.org/maven2/com/snowplowanalytics/scala-maxmind-iplookups_2.11/0.2.0/scala-maxmind-iplookups_2.11-0.2.0.pom
Downloading https://repo1.maven.org/maven2/com/snowplowanalytics/scala-maxmind-iplookups_2.11/0.2.0/scala-maxmind-iplookups_2.11-0.2.0.pom.sha1
Downloading http://maven.snplow.com/releases/com/snowplowanalytics/scala-maxmind-iplookups_2.11/0.2.0/
Downloading http://maven.twttr.com/com/snowplowanalytics/scala-maxmind-iplookups_2.11/0.2.0/scala-maxmind-iplookups_2.11-0.2.0.pom
Downloading http://maven.twttr.com/com/snowplowanalytics/scala-maxmind-iplookups_2.11/0.2.0/scala-maxmind-iplookups_2.11-0.2.0.pom.sha1
[trace] Stack trace suppressed: run last *:coursierResolution for the full output.
[error] (*:coursierResolution) coursier.ResolutionException: Encountered 1 error(s) in dependency resolution:
[error]   com.snowplowanalytics:scala-maxmind-iplookups_2.11:0.2.0:
[error]     not found:
[error]       /usr/local/.ivy2/local/com.snowplowanalytics/scala-maxmind-iplookups_2.11/0.2.0/ivys/ivy.xml
[error]       https://repo1.maven.org/maven2/com/snowplowanalytics/scala-maxmind-iplookups_2.11/0.2.0/scala-maxmind-iplookups_2.11-0.2.0.pom
[error]       http://maven.snplow.com/releases/com/snowplowanalytics/scala-maxmind-iplookups_2.11/0.2.0/
[error]       http://maven.twttr.com/com/snowplowanalytics/scala-maxmind-iplookups_2.11/0.2.0/scala-maxmind-iplookups_2.11-0.2.0.pom
```

And indeed the `http://maven.snplow.com/releases/com/snowplowanalytics/scala-maxmind-iplookups_2.11/0.2.0/` URL doesn't lead anywhere. But if I append `scala-maxmind-iplookups_2.11-0.2.0.pom` to it, it leads to an actual [file](http://maven.snplow.com/releases/com/snowplowanalytics/scala-maxmind-iplookups_2.11/0.2.0/scala-maxmind-iplookups_2.11-0.2.0.pom).

After running the above command the following files appear under 
```.coursier/cache/v1/http/maven.snplow.com/releases/com/snowplowanalytics/scala-maxmind-iplookups_2.11/0.2.0```:
```
.scala-maxmind-iplookups_2.11-0.2.0.pom.checked     
.scala-maxmind-iplookups_2.11-0.2.0.pom.sha1.checked
scala-maxmind-iplookups_2.11-0.2.0.pom              
scala-maxmind-iplookups_2.11-0.2.0.pom.sha1
```

The problem disappears when reverting back to coursier `1.0.0-M14-9`.

Any help would be appreciated,
Thanks.

**Comment by sgrankin** [comment]
Seeing this as well with the confluent.io maven repo; fails starting with M15-2 through -5.
Minimum repro at https://gist.github.com/sgrankin/08b6ba85e7e800337f02890ea38b63fb

**Comment by sgrankin** [comment]
M15-7 fixes this. Thanks!

**Comment by ncreep2** [comment]
Seems to be fixed with `1.0.0-RC1` as well, thanks.

**Comment by ncreep** [comment]
Seems to be fixed with `1.0.0-RC1` as well, thanks.
